src/monero_ux_nano.c

```diff
@@ -606,21 +606,21 @@ int ui_menu_any_pubaddr_display(unsigned int value __attribute__((unused)), unsi
     switch (G_monero_vstate.disp_addr_mode) {
         case 0:
         case DISP_MAIN:
-            memcpy(ADDR_TYPE, "Main", 4);
-            memcpy(ADDR_MAJOR, "Major: 0", 8);
-            memcpy(ADDR_MINOR, "minor: 0", 8);
+            strncpy(ADDR_TYPE, "Main", 4);
```

Comment:
I don't get the point of the `strncpy` here.
The size value is hardcoded and match `sizeof(src) -1`. It will always fit and the `\0` will never be included...

IMO `memcpy(ADDR_TYPE, "Main", sizeof("Main");` would be better: enforce a `\0` and doesn't mislead with the value which as nothing to do with the dest buffer length (as expected in a classical `strncpy` usage).